README.md

```diff
@@ -130,7 +130,7 @@ pathname            | The URL namespace that Primus can own     | `/primus`
 parser              | Message encoder for all communication     | `JSON`
 transformer         | The transformer we should use internally  | `websockets`
 plugin              | The plugins that should be applied        | `{}`
-timeout             | The heartbeat timeout                     | `35000`
+timeout             | Interval at which heartbeats are sent     | `30000`
```

Comment:
Not sure if it makes sense to change the option name and use something more explicative